scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/soic.yaml

```diff
@@ -432,7 +432,7 @@ SOIC-8_7.5x5.85mm_P1.27mm:
 
 SOIC-14_3.9x8.7mm_P1.27mm:
   #round to two significant digits to comply with old name
-  custom_name_format: SOIC-{pincount:d}_{size_x:.2g}x{size_y:.2g}mm_P{pitch:g}mm
+  custom_name_format: SOIC-{pincount:s}_{size_x:.2g}x{size_y:.2g}mm_P{pitch:g}mm
```

Comment:
Why did you change all of those?
**edit**: I see that you commented on that. My first guess is that this is an unintended changed in the generator.

Reply:
Yeah, I can undo this change if you want, but then the script crashes on generating `soic.yaml`.

Reply:
FYI: We are discussing this issue in https://github.com/pointhi/kicad-footprint-generator/pull/649